src/style.css

```diff
@@ -1,73 +1,126 @@
+/* ------------ youtube.com/CodeExplained ------------ */
 body {
-  background-color: rgb(241, 233, 233);
+  padding: 0;
```

Comment:
- [ ] A function for deleting a task (removing an element from the array) is implemented. ([this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/sneak_peek.md#:~:text=Your%20task%20is%20to%20build%20a%20website%20that%20displays%20a%20list%20that%20looks%20and%20behaves%20like%20the%20part%20of%20minimalist%20project%20capture%20in%20the%20video%20below.)) ❌

It's removing all items when I click the refresh button. But it's not meant to be. If I click `trashcan` you need to remove item form list as well as from the localstore.